### DISCUSSION TITLE
Support for Jumpcloud?

### DISCUSSION THREAD

**Discussion by matanyall** [infrahq/infra#3565] [Q&A]
I use Jumpcloud to manage all of my users and devices. Is there support coming soon/ on the roadmap?

**Comment by mchiang0610** [comment]
We currently support OIDC, and either JumpCloud will support OIDC, which seems like it will happen ( https://github.com/tailscale/tailscale/issues/794#issuecomment-1063270591 ) or we have to wait until we support SAML. 

Would love to hear more about your use case? How many users do you manage with JumpCloud or if you have another identity provider already in use that can support OIDC?
